digital_land/phase/map.py

```diff
@@ -6,8 +6,13 @@
 normalise_pattern = re.compile(r"[^a-z0-9-_]")
 
 
+def replace_underscores(name: str):
```

Comment:
This looks right but again shouldn't be in this PR. also I'm not sure we need another function to remove underscores when it's one line it could just be in the normalise function with an extra test